teps/0050-ignore-task-failures.md

```diff
@@ -187,6 +194,222 @@ control over the task definitions but may desire to ignore a failure and continu
 
   ![Jenkins Dashboard](images/0050-jenkins-dashboard-with-failure-stage.png)
 
+## Proposal
+We propose a new filed ```OnFail``` to the [PipelineTask](https://github.com/tektoncd/pipeline/blob/main/docs/pipelines.md#adding-tasks-to-the-pipeline) definition.
+
+```go
+type PipelineTask struct {
+	Name string `json:"name,omitempty"`
+
+	//...
+
+	// OnFail defines the termination behavior of a pipeline when the task is failed
+	// can be set to [ continue | stopAndFail ]
+	OnFail OnFailType `json:onFail, "omitempty"`
```

Comment:
please consider supporting params as well 🙏 

```
onFail: $(params.CONTINUE)
```

Reply:
This is definitely a feature nice to have, but I feel this is not in the scope of this TEP which is to implement a task failure strategy? I can create an issue to track this request, and work on it separately after this TEP is implemented. 😄

Reply:
@QuanZhang-William you can add a "future work" section describing supporting parameters

Reply:
SG! Will do!